languages/python/config.json

```
@@ -8,7 +8,14 @@
     "indent_size": 4
   },
   "exercises": {
-    "concept": [],
+    "concept": [
+      {
+        "slug": "strings",
+        "uuid": "",
+        "concepts": ["strings-methods"],
```

In other tracks, we're using the `strings-basic` concept for the strings exercise. This is no formal requirement, but we do want to strive towards using cross-track concepts.

```suggestion
        "concepts": ["strings-basic"],
```

Additionally, does "strings-methods" cover what is being taught completely? I think the student is also taught how to define strings for example, which fits "strings-methods" less well I feel.

---

Let's go with `strings-basic` ... Python has _a lot_ of methods on the **str** type and this will barely cover any of them. We'll need a follow up exercise to cover more.

---

Probably. The idea of the `strings-basic` exercise in C# is that the student will learn where to look for the methods/functions that are available. We then don't have to teach all the individual functions, but only the skill to find them.

---

Is the `configlet` the item I need to generate the UUID here? Still learning my way around the various projects.